go/vt/vttablet/tabletserver/txthrottler/tx_throttler.go

```diff
@@ -114,10 +113,22 @@ func tryCreateTxThrottler(config *tabletenv.TabletConfig, topoServer *topo.Serve
 		return nil, err
 	}
 
-	// Clone tsv.TxThrottlerHealthCheckCells so that we don't assume tsv.TxThrottlerHealthCheckCells
-	// is immutable.
-	healthCheckCells := make([]string, len(config.TxThrottlerHealthCheckCells))
-	copy(healthCheckCells, config.TxThrottlerHealthCheckCells)
+	var healthCheckCells []string
+	if len(config.TxThrottlerHealthCheckCells) > 0 {
+		// Clone tsv.TxThrottlerHealthCheckCells so that we don't assume tsv.TxThrottlerHealthCheckCells
+		// is immutable.
+		healthCheckCells = make([]string, len(config.TxThrottlerHealthCheckCells))
+		copy(healthCheckCells, config.TxThrottlerHealthCheckCells)
```

Comment:
Nit: can be done in one line, and I think with less clutter, via:
```suggestion
		healthCheckCells = append(healthCheckCells, config.TxThrottlerHealthCheckCells...)
```